lib/parser/file_group.ex

```diff
@@ -92,4 +99,35 @@ defmodule Thrift.Parser.FileGroup do
     other
   end
 
+
+  def dest_module(file_group, %Struct{name: name}) do
+    dest_module(file_group, name)
+  end
+
+  def dest_module(file_group, name) do
+    [thrift_module, struct_name] = name
+    |> Atom.to_string
+    |> String.split(".")
+    |> Enum.map(&String.to_atom/1)
+
+    case file_group.ns_mappings[thrift_module] do
+      nil ->
+        Module.concat(Elixir, struct_name)
+      namespace = %Namespace{} ->
+        namespace.path
+        |> String.split(".")
+        |> Enum.map(&Macro.camelize/1)
+        |> Enum.join(".")
```

Comment:
There's duplication between this code and Thrift.Generator.Models.namespace that we should consolidate.